openpype/tools/traypublisher/window.py

```diff
@@ -90,6 +109,17 @@ def __init__(self, publisher_window):
 
     def showEvent(self, event):
         self._projects_model.refresh()
+
+        setting_registry = TrayPublisherRegistry()
+        project_name = setting_registry.get_item("project_name")
+        if project_name:
+            index = self._projects_model.get_index(project_name)
+            if index:
```

Comment:
There is missing conversion to proxy index. (Changes are based on [these changes](https://github.com/pypeclub/OpenPype/pull/3867/files#diff-1ecdf4cf8aaa62809627f8d57a300cb7a04414f370adaa27405e39bb6712d429R96) )
```suggestion
            index = None
            src_index = self._projects_model.get_index(project_name)
            if src_index is not None:
                index = self._projects_proxy.mapFromSource(src_index)
            if index:
```

Reply:
Updated